src/venue/service.js

```diff
@@ -146,6 +146,27 @@ angular
       };
 
       /*==============================================================*
+      /* Info
+      /*==============================================================*/
+
+
+      VenueService.prototype.listInfo = function listInfo(venueId) {
```

Comment:
For GET functions I like to always allow a second (optional)  `options` object so I can add things to the query if needed. This is a good example in FeedService:
```
FeedService.prototype.listUserFeed = function(userId, options) {
        if (!userId) throw new Error('FeedService.listUserFeed - userId is required');
        options = options || {};

        options.userId = userId;

        return Feed.listUserFeed(options).$promise;
      };
```